src/theme/Admonition/index.js

```diff
@@ -43,7 +43,7 @@ function InfoIcon() {
     </svg>
   );
 }
-function CautionIcon() {
+function WarningIcon() {
```

Comment:
There was an error for `WarningIcon` not existing. `CautionIcon` was not used, so I made an assumption that this was the one that it was supposed to be?